src/core/structures/PDFAcroFormField.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  PDFArray,
+  PDFDict,
+  PDFName,
+  PDFNumber,
+  PDFObject,
+  PDFString,
+} from 'src/core';
+
+const acroFormFieldTypes: Array<PDFObject | undefined> = [
```

Comment:
TypeScript should be able to automatically infer the type of this array as `PDFRef[]`, so I don't think this type annotation is necessary at all. However, if it actually is necessary for some reason, we should refine it to `const acroFormFieldTypes: PDFRef[]`.

Reply:
Updated